Some cleaning up of the article templates to make it forkable

mrshirts opened the pull request:
* superscripted references
* reordered the README.md to be something more like instructions for forking
* added stub your.README.md the authors can move
* Added options to the livecoms class so that we can have different formatting for different types of documents.  Right now, 
* Turned the template to just a best practices guide template.
* Moved the figures to a separate folder.

davidlmobley commented on your.README.md:
```diff
@@ -0,0 +1,2 @@
+Will put more information here later.  This will be the readme where
```
If we're going to set this up for forking, probably people are going to start doing this right away, so shouldn't we put the info we want them to have in their README here before merging? 

We will at least want markdown-format header (title) and subheaders and dummy text so they can easily see how to do those.

Obvious things to have authors put here:
- Appropriate markdown format header/title
- Brief explanation of what their focus is in this repository/work
- List of authors with contributions (to be updated as they go)
- List of acknowledgments with contributions (to be updated as they go)

Things we should probably put here:
- Very brief summary of this model of paper writing (link to "paper writing as code development?)
- Encouragement to community to use Issues button to make suggestions, for discussion, feedback, etc.
- Note that specific changes can be proposed via pull requests (eventually this would have a link to info on how to use Git/GitHub for this) 
- Maybe link to LiveCoMS, one or two sentence summary of what it is and how this repo (their repo) relates? 
- List of released versions saying "none yet"
- Changelog ("none yet")

mrshirts replied:
Are you imaging a changelog that get automatically updated somehow, or we expect them to provide a summary of changes?

davidlmobley replied:
@mrshirts - never seen a useful way to automatically update a changelog, [because of this](https://xkcd.com/1296/) , but if you have one I'd love to hear it. No, this would be the responsibility of a maintainer who is interested in keeping people up to date on what is happening. Presumably if they don't want to do it they should remove it.

mrshirts replied:
OK, just checking.  I added a pass at this to the pull request when you have time to review.

davidlmobley commented on information_for_authors.md:
```diff
@@ -0,0 +1,3 @@
+**To get started with our template, clone this repository or click the "clone or download button" and download a zip file. Then navigate to the `templates` directory, copy the files to where you will be working, and get started!**
+
+**After forking, edit `your.README.md` and move it to `READ.md`, and you are set to write the article in your own repostitory. You can also delete the `example.tex` files that don't match your article type.
```
Maybe add a link to Markdown format info? A naive author may not even know these are Markdown format (I've had people contribute to mine who think these are just plain text).

mrshirts commented:
OK, take a look now.  It doesn't have to be finalized at this point -- we need to really collect some more information before it's truly usable.

davidlmobley commented on README.md:
```diff
@@ -1,20 +1,5 @@
-
 ## LaTeX and GitHub
 
 LaTeX is a powerful typesetting program that is very commonly used in the mathematics and physics community, and increasingly in the computational sciences. We have a provide a LaTeX template for LiveCoMS in order to make it easy to have a pleasing, consistent visual style. 
```
"We have a provide a LaTeX template" -> "We provide a LaTeX template".

davidlmobley replied:
(Also, and it won't let me comment on this for some reason, there is an instance just below where "formatting" is spelled as "formating".)

mrshirts replied:
Fixed.

davidlmobley commented on README.md:
```diff
@@ -1,20 +1,5 @@
 # Preparing LaTex articles using GitHub for the Living Journal of Computational Molecular Sciences ([LiveCoMS](http://www.livecomsjournal.org/))
 
-## About the journal
```
Seems like we should be replacing this with some kind of info on what this repository is about, e.g. `## This repository provides templates for LiveCoMS articles`, then under it a brief explanation of what to do with the repository (fork it!), perhaps also with a link to an explanation of the journal model.

Remember, this README.md is the MAIN THING people see when they come to github.com/livecomsjournal so it will help if we can tell them what they are supposed to find here and what they should do with it.

davidlmobley replied:
(For example, THIS displays on the main page but it contains no link to the `information_for_authors.md` file, which has the actual instructions.)

I know we discussed this before, but maybe it is better to just put the `information_for_authors.md` contents in THIS file and have no separate file? 

Or, maybe what we said we wanted to do was:
1) Make THIS `README.md` be just the info about (a) what our templates repository is for and (b) what authors should do with it (so including what's currently in `information_for_authors.md`) -- that is, stuff which should get wiped out after they fork
2) Make `information_for_authors.md` contain everything they will STILL need after they fork
I'm forgetting.

If we did that, then we would make sure both this `README.md` and the `your.README.md` both link to the `information_for_authors.md` so the organization of the repo is clear both before AND after they follow our instructions to set up their paper.

Main point: Right now the organization of the repo is not clear before OR after forking -- readers are not directed to `information_for_authors.md` anywhere, but they have to get there to find the instructions they need to follow to fork. Who will tell them to look there?

davidlmobley commented on information_for_authors.md:
```diff
@@ -0,0 +1,11 @@
+
+## After Forking
+
+Edit `your.README.md` and move it to `READ.md`, and you are set to write the article in your own repostitory. 
```
"repository" not "repostitory"

mrshirts replied:
fixed.

davidlmobley commented on information_for_authors.md:
```diff
@@ -0,0 +1,11 @@
+## Getting started
+
+To get started with our template, clone [this repository](https://github.com/livecomsjournal/article_templates) by clicking the "clone or download button" and download a `.zip` file. 
+Then navigate to the `templates` directory, copy the files to where you will be working, and get started!
```
These instructions seem unclear -- we have an "after forking" section but we don't tell them to fork. And are we proposing two models here? (Getting started without forking, or getting started via forking?)

It seems like we want to just write step-by-step instructions. I think these should also include (a) first forking, then (b) retrieving exactly the files they want (e.g. the specific template for best practices documents or some such), (c) removing unnecessary files, (d) moving the `your.README.md` as you describe, then (e) editing it.

Probably they should also get a default license when they fork; we should make sure it is the one we want as default (CC-BY), but add a note pointing out they can update if they want...

mrshirts commented:
I've addressed most of the comments, though perhaps not perfectly.  I'm going to merge this in so people can start to provide feedback for the templates.
